projects/sticks.md

```diff
@@ -0,0 +1,38 @@
+# Sticks
```

Comment:
move to e.g. `projects/python/sticks.md`

Reply:
Done!